ai_agents/devable_research_agent/lib/github.py

```
@@ -0,0 +1,305 @@
+            # Normalize OSS Insight score to 0-1 for relevance boost
+            norm_score = total_score / max_score if max_score > 0 else 0
+            relevance = min(1.0, relevance + norm_score * 0.15)
+
+            all_items.append(TrackerItem(
+                id=f"GH-TREND-{repo_name}",
+                title=repo_name,
+                summary=description[:300] if description else f"Trending repo with {stars} stars",
+                entity=entity_label,
+                source=SOURCE_GITHUB,
+                source_url=f"https://github.com/{repo_name}",
+                source_label=f"GitHub Trending (OSS Insight)",
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)